### PR TITLE
"Everyday Accessibility" post

### PR DESCRIPTION
**Pull request by MicheleAWilliams-A11y**
This is in regards to the idea discussed in the comments of #12. 
Caveats:
-"Everyday Accessibility" is a tentative title - happy to change.
-I can't upload documents so I copied my MD content into this - hope it works.
-Since I can't upload here, I'll need a means to send my image files; just let me know.
-I mimicked the code I saw to create figure captions along with alt text; hoping there's no mistakes.
-I wasn't sure of all the front matter so I left some fields blank. For the "category", I think it's part of "Quick Tips" but didn't know how that should be typed.
-Line 28 ("When you write") should have "clapping hands" emojis. So far it's copying the images but not sure how to make sure that shows in the final version so heads up.

### CODE REVIEW

**Review by mxmason — COMMENTED** [review]
Thanks for working on this, @MicheleAWilliams-A11y, and thank you for your patience with getting this rendered in a way that you like. I like that you're covering some small but meaningful steps that everyone can take, and relating those steps to things we do every day. My notes are mainly editorial in nature.

Broadly: Can we convert your bold topic intros (like "Don’t share just images of text") into `h3`s? I think that would help make the content easier to parse. 

Specifically: I have left some discrete comments for your consideration! If you like a suggested change, you can accept it by clicking "Add suggestion to batch". When you have collected all of the suggestions you'd like, you can click "Commit suggestions". The GitHub UI will prompt you to write a commit message, the same as you have done for your commits so far. If you have questions or would like to discuss my suggestions further, you can comment on them and we can dig in!

You can read more about the suggested changes feature (with pictures!) in [GitHub's documentation on it](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request).

File: src/posts/2021-06-14-Everyday_Accessibility.md
Comment (MicheleAWilliams-A11y) [reply src/posts/2021-06-14-Everyday_Accessibility.md:29]: Thought about this for sure. I found a few possibilities: 

- RNIB article - it's similar to this one but lists out how to add descriptions on Twitter, FB, and Instragram: https://www.rnib.org.uk/rnibconnect/technology/making-your-social-media-accessible
- Symphony - not familiar with the company but this is a succinct article with pictures and a date of 2020 (though I'm not sure how accurate the Twitter one is): https://symphonyagency.com/alt-text-for-social-media/

Either way it's still worth telling people to search for the "How To" so they are always current and look throughout each platform (like how these articles don't mention TikTok, etc.).

File: src/posts/2021-06-14-Everyday_Accessibility.md
Comment (MicheleAWilliams-A11y) [reply src/posts/2021-06-14-Everyday_Accessibility.md:85]: That's what I was trying to say with "rapidly moving content" without repeating "flashing content" again. The guideline should read that you _can't_ do flashing content.
Maybe we can change the heading to be more direct? E.g., "Do not have any flashing content".
Then the paragraph can say: "Content that flashes on the screen more than 3 times per second is dangerous as it can trigger a seizure in some viewers. Use rapid movement sparingly in your videos and ensure nothing flashes."

File: src/posts/2021-06-14-Everyday_Accessibility.md
Comment (MicheleAWilliams-A11y) [reply src/posts/2021-06-14-Everyday_Accessibility.md:98]: How about "...reach out to the company to ask about their accessibility policy and roadmaps via the company's customer support channels."

**Review by MicheleAWilliams-A11y — COMMENTED** [review]
Agree to making things headings as well but will do that once I know my other comments went through. Thanks!

File: src/posts/2021-06-14-Everyday_Accessibility.md
Comment (mxmason) [reply src/posts/2021-06-14-Everyday_Accessibility.md:85]: This one is my bad: I did not parse your intro text correctly. Looking at this again, I think we agree. I think whatever language you settle on should emphasize having no flashing content at all, but if you must, keep flashes brief, etc etc.

File: src/posts/2021-06-14-Everyday_Accessibility.md
Comment (mxmason) [reply src/posts/2021-06-14-Everyday_Accessibility.md:98]: I like that!

File: src/posts/2021-06-14-Everyday_Accessibility.md
Comment (ericwbailey) [inline src/posts/2021-06-14-Everyday_Accessibility.md:14]: Love this. Direct and to the point.

File: src/posts/2021-06-14-Everyday_Accessibility.md
Comment (ericwbailey) [inline src/posts/2021-06-14-Everyday_Accessibility.md:17]: Can we add a bridge sentence here that queues up the content in the subheadings? I've been told that without it there can be confusion if content is missing/failed to load, or the heading structure is broken somehow.

File: src/posts/2021-06-14-Everyday_Accessibility.md
Comment (ericwbailey) [inline src/posts/2021-06-14-Everyday_Accessibility.md:19]:
```suggestion
### Use capital letters in hashtags

Punctuation helps text-to-speech tools like screen readers speak more accurately, and helps with readability overall. This includes in hashtags, where capital letters help distinguish words.
```

Adding an empty space between the heading and the following sentence so the heading will render properly.

File: src/posts/2021-06-14-Everyday_Accessibility.md
Comment (ericwbailey) [inline src/posts/2021-06-14-Everyday_Accessibility.md:24]:
```suggestion
### Don’t share just images of text

Images of text (such as your favorite quote or saying) can be difficult for people with visual impairments to read and don’t allow people with reading difficulties to leverage text-to-speech read-aloud features. When posting these kinds of images, also write out the text in your post.
```

Adding another space between heading and content here.

File: src/posts/2021-06-14-Everyday_Accessibility.md
Comment (ericwbailey) [inline src/posts/2021-06-14-Everyday_Accessibility.md:29]: 💯

File: src/posts/2021-06-14-Everyday_Accessibility.md
Comment (ericwbailey) [inline src/posts/2021-06-14-Everyday_Accessibility.md:32]:
```suggestion
### Use “alt text” features to describe images

Pictures are significant in social media, but also inaccessible to visually impaired readers unless they have a text-based description. When uploading, find the “image description” or “alternative/alt text” feature in the software to write a description. For sample instructions, review [Alt Text for Social Media](https://symphonyagency.com/alt-text-for-social-media/). 
```

File: src/posts/2021-06-14-Everyday_Accessibility.md
Comment (ericwbailey) [inline src/posts/2021-06-14-Everyday_Accessibility.md:42]:
```suggestion
### Limit your emojis

Emojis are great for emphasis and playfulness, but too many and in unconventional placements can be distracting and hard to read. Keep your emojis to the minimum needed to make your point, group them together, and move them towards the end of any text.
```

Comment (ericwbailey) [reply src/posts/2021-06-14-Everyday_Accessibility.md:42]: >distracting and hard to read

Do you think it's worth mentioning the announcement clutter created in screen reader announcements when emojis are over-applied?

File: src/posts/2021-06-14-Everyday_Accessibility.md
Comment (ericwbailey) [inline src/posts/2021-06-14-Everyday_Accessibility.md:50]:
```suggestion
### Use a program’s built-in styling features

Assistive technology (AT) doesn’t interpret a page or document like people do with their eyes; it reads the underlying code markup. This means simply making something _look_ a certain way falls apart when using AT. When formatting a document or presentation, use the style buttons/options in the program to provide the needed markup; for instance, use the “bulleted list” button rather than just a dash (-) to format a list.
```

Comment (ericwbailey) [reply src/posts/2021-06-14-Everyday_Accessibility.md:50]: Can we turn the sentence with a semicolon into two separate sentences/thoughts?

File: src/posts/2021-06-14-Everyday_Accessibility.md
Comment (ericwbailey) [inline src/posts/2021-06-14-Everyday_Accessibility.md:53]: Finally, the Ribbon is good for something 🤣

File: src/posts/2021-06-14-Everyday_Accessibility.md
Comment (ericwbailey) [inline src/posts/2021-06-14-Everyday_Accessibility.md:58]:
```suggestion
### Include document structure

Headings (or titles on slides) provide a great way for everyone to preview, skim, and search content. Structure is particularly helpful for blind readers who want to get an overview of the document before reading line-by-line. Include headings throughout a document and titles on every slide, and remember to use the actual “Heading” and “Title” features of the program, not simply stylized text.
```

File: src/posts/2021-06-14-Everyday_Accessibility.md
Comment (ericwbailey) [inline src/posts/2021-06-14-Everyday_Accessibility.md:66]:
```suggestion
### Add alt text to images

Just like with social media, you can add alt text descriptions to images in documents as well. You will typically see an input box to add descriptions grouped with other options when you insert the image.
```

File: src/posts/2021-06-14-Everyday_Accessibility.md
Comment (ericwbailey) [inline src/posts/2021-06-14-Everyday_Accessibility.md:74]:
```suggestion
### Use accessibility checkers

Microsoft products (namely Word and PowerPoint) come with a “Check Accessibility” option. Use this to double-check you’ve addressed key accessibility needs.
```

File: src/posts/2021-06-14-Everyday_Accessibility.md
Comment (ericwbailey) [inline src/posts/2021-06-14-Everyday_Accessibility.md:84]: Can we add a sentence about who or how this helps?

File: src/posts/2021-06-14-Everyday_Accessibility.md
Comment (ericwbailey) [inline src/posts/2021-06-14-Everyday_Accessibility.md:83]:
```suggestion
### Have clear audio

```

File: src/posts/2021-06-14-Everyday_Accessibility.md
Comment (ericwbailey) [inline src/posts/2021-06-14-Everyday_Accessibility.md:89]:
```suggestion
### Reduce background music

Instrumental music can enhance content’s quality, but not when it reduces the ability to hear the main speaker. When using background music, keep it to a minimum (such as during an intro) or, if used throughout, ensure it is very low.
```

File: src/posts/2021-06-14-Everyday_Accessibility.md
Comment (ericwbailey) [inline src/posts/2021-06-14-Everyday_Accessibility.md:94]:
```suggestion
### Provide text with multimedia

As with images, video and audio can be engaging but also inaccessible without a text equivalent for people who are deaf-blind, having sensory processing disorders, or have hearing impairments. Having media content written out as text is vital for inclusiveness. Text equivalents can include transcripts that type out spoken dialogue, or an article that accompanies a video (as in a video that supplements a news article).
```

File: src/posts/2021-06-14-Everyday_Accessibility.md
Comment (ericwbailey) [inline src/posts/2021-06-14-Everyday_Accessibility.md:98]:
```suggestion
### Do not have any flashing content

```

File: src/posts/2021-06-14-Everyday_Accessibility.md
Comment (ericwbailey) [inline src/posts/2021-06-14-Everyday_Accessibility.md:99]:
```suggestion
Content that flashes on the screen more than [3 times per second](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-three-times.html) is dangerous as it can trigger a seizure in some viewers. Use rapid movement sparingly in your videos and ensure nothing flashes.
```

You don't have to accept this suggestion, but I oftentimes find myself needing to invoke it for arguments involving animation. People get emotional about removing animated content, even if it's dangerous, so it might be helpful for the reader to have a resource to use if things get dicey.

File: src/posts/2021-06-14-Everyday_Accessibility.md
Comment (ericwbailey) [inline src/posts/2021-06-14-Everyday_Accessibility.md:105]:
```suggestion
### Seek accessibility features

```

File: src/posts/2021-06-14-Everyday_Accessibility.md
Comment (ericwbailey) [inline src/posts/2021-06-14-Everyday_Accessibility.md:111]: Can we use an example other than Typeform? They have poor AT support and have publicly stated that improving the accessibility of their platform isn't a priority.

File: src/posts/2021-06-14-Everyday_Accessibility.md
Comment (ericwbailey) [inline src/posts/2021-06-14-Everyday_Accessibility.md:113]:
```suggestion
### Ask about accessibility

```

File: src/posts/2021-06-14-Everyday_Accessibility.md
Comment (ericwbailey) [inline src/posts/2021-06-14-Everyday_Accessibility.md:118]:
```suggestion
### Seek to learn more 

```

File: src/posts/2021-06-14-Everyday_Accessibility.md
Comment (ericwbailey) [inline src/posts/2021-06-14-Everyday_Accessibility.md:121]: 🧡

**Review by ericwbailey — CHANGES_REQUESTED** [review]
This is great, @MicheleAWilliams-A11y! I'm really excited to publish it. Overall feedback:

1. It may look like a lot of feedback, but 90% of it is formatting headings. You should be able to click the "Accept suggestion" button on the review to incorporate the change automatically.
1. A couple of asks for examples to help the reader understand the why.
1. A request to use an example other than Typeform. I can help you source an alternative, if you'd like.

Also, could you send us a message via our [Contact Form](https://www.a11yproject.com/contact/)? We can coordinate payment from there.

File: src/posts/2021-06-14-Everyday_Accessibility.md
Comment (MicheleAWilliams-A11y) [reply src/posts/2021-06-14-Everyday_Accessibility.md:17]: Yes, @mxmason and I were just discussing this. Initially I thought I could sneak by when I had these as numbered lists but now that they're headings I'll add a little something. I just hate when it's sounds too redundant (e.g., waste of reading time) so I'll see what _useful_ content I can come up with so it's not drab...

File: src/posts/2021-06-14-Everyday_Accessibility.md
Comment (MicheleAWilliams-A11y) [reply src/posts/2021-06-14-Everyday_Accessibility.md]: I'll be sure to catch these throughout; something to note for content authors. In my writing software (Ulysses) the extra spaces aren't needed when the content is rendered (in the templates they provide) so I didn't know to do it until later.

File: src/posts/2021-06-14-Everyday_Accessibility.md
Comment (MicheleAWilliams-A11y) [reply src/posts/2021-06-14-Everyday_Accessibility.md:45]: Sure, since it's related to the example as well (assuming I keep that example). I'll rephrase that first sentence to say something like "it's cluttered with text-to-speech and hard to read visually". That should address both use cases.

File: src/posts/2021-06-14-Everyday_Accessibility.md
Comment (MicheleAWilliams-A11y) [reply src/posts/2021-06-14-Everyday_Accessibility.md:53]: Okay, sure. So we'll say something like, "Assistive technology (AT) doesn't interpret a page or document like people do with their eyes. AT is reading the underlying code markup which means simply making something _look_ a certain way is not sufficient." I'll update this and put it in when I upload my other changes to see if it flows better.

File: src/posts/2021-06-14-Everyday_Accessibility.md
Comment (MicheleAWilliams-A11y) [reply src/posts/2021-06-14-Everyday_Accessibility.md:56]: You're a mess...

File: src/posts/2021-06-14-Everyday_Accessibility.md
Comment (MicheleAWilliams-A11y) [reply src/posts/2021-06-14-Everyday_Accessibility.md]: Great catch! Will add.

File: src/posts/2021-06-14-Everyday_Accessibility.md
Comment (MicheleAWilliams-A11y) [reply src/posts/2021-06-14-Everyday_Accessibility.md:108]: Yikes! Okay, let me see if I can rephrase to have a better link text and I'll also point to v2.1 just so folks are dropped into the latest guidelines.

File: src/posts/2021-06-14-Everyday_Accessibility.md
Comment (MicheleAWilliams-A11y) [reply src/posts/2021-06-14-Everyday_Accessibility.md:134]: This was literally my idea to put even before I thought "Hey, maybe they might want this article!" LOL

File: src/posts/2021-06-14-Everyday_Accessibility.md
Comment (MicheleAWilliams-A11y) [reply src/posts/2021-06-14-Everyday_Accessibility.md:122]: [Curse word] Figured as much. Yup, I'll take a gander now...

**Comment by MicheleAWilliams-A11y** [comment]
Working through the changes from Eric and also per discussion with EJ - will be updating soon!

**Comment by MicheleAWilliams-A11y** [comment]
I used the "Edit file" feature to make updates. I hope they come through correctly [fingers crossed]. Also, I need to upload a new photo - I'll email to @mxmason like I did last time. 

I think this will flow a lot better - thanks for your help and patience! This has been a great collaboration thus far!

**Review by MicheleAWilliams-A11y — COMMENTED** [review]
Trying to get the "Changes Requested" status to go away... Thought I wasn't supposed to hit this "Review Changes" button but seems this is how to clear things. I'm going to say "Comment" since I didn't accept all the changes from @ericwbailey. More GitHub fun I see...

File: src/posts/2021-06-14-Everyday_Accessibility.md
Comment (ericwbailey) [inline src/posts/2021-06-14-Everyday_Accessibility.md:133]: The preview is showing a broken image here. I think we need to delete `Accessible_Typeforms.png `, and then add `Zoom_A11y_Page.png` so it loads properly.

**Review by ericwbailey — CHANGES_REQUESTED** [review]
@MicheleAWilliams-A11y This is so close, and I'm really excited to publish it. Once we fix the missing image it should be good to go 🎉

>Trying to get the "Changes Requested" status to go away... Thought I wasn't supposed to hit this "Review Changes" button but seems this is how to clear things. I'm going to say "Comment" since I didn't accept all the changes from @ericwbailey. More GitHub fun I see...

Noted. The way that works is there's a back-and-forth between the maintainers of the site and the person requesting code changes via a Pull Request. It's up to maintainers to deal with the status, as we're the ones who have final say over whether code gets incorporated or not.

File: src/posts/2021-06-14-Everyday_Accessibility.md
Comment (MicheleAWilliams-A11y) [reply src/posts/2021-06-14-Everyday_Accessibility.md:133]: Yay! Yes, I think I emailed that to @mxmason because I don't have access/permissions to upload. If you want to message me a place to email it I can do that ASAP so we can get rolling! This was a great collaboration!

Comment (ericwbailey) [reply src/posts/2021-06-14-Everyday_Accessibility.md:133]: Just pushed it up!

**Review by ericwbailey — APPROVED** [review]

**Pull request merged**
